app/models/hearing_summary.rb

```diff
@@ -19,6 +19,10 @@ def hearing_days
     body['hearingDays'].map { |hearing_day| hearing_day['sittingDay'] }
   end
 
+  def hearing_resulted?
+    Hearing.find_by(id: id) != nil
```

Comment:
```suggestion
    Hearing.find_by(id: id).present?
```
This would be better served by a `present?` check perhaps.